manifests/init.pp

```diff
@@ -243,8 +243,8 @@
   Integer[0] $api_service_worker_count = min(4, $facts['processors']['count']) + 1,
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
-  Integer[0] $api_service_worker_max_requests = 50,
-  Integer[0] $api_service_worker_max_requests_jitter = 30,
+  Integer[0] $api_service_worker_max_requests = 800,
+  Integer[0] $api_service_worker_max_requests_jitter = 100,
```

Comment:
These seem reasonable to me, but they're just spitballs.  Probably 4,000 and 500 would be fine too.